### DISCUSSION TITLE
when you have # in a value

### DISCUSSION THREAD

**Discussion by tbratsas** [rafael-figueiredo-alves/DotEnv4Delphi#7] [General]
There is this  function RemoverComentario when is called it removes from a value the # for example .env file DBPASSWORD=E#6vK7z it gives me the value E6vK7z

**Comment by rafael-figueiredo-alves** [comment]
Thanks @tbratsas . I'll be checking the code and an updated version will be available soon.

**Comment by rafael-figueiredo-alves** [comment]
@tbratsas , I've fixed the issue on the latest release, version 1.5.0. Please test it as soon as you can. I'll be waiting for your feedback.

**Reply by tbratsas** [reply]
@rafael-figueiredo-alves Everything works fine. Thank you !

**Reply by rafael-figueiredo-alves** [reply]
You're welcome. And thank you for bringing it up to me.
